editor/frontend/src/CreateElementTool.ts

```diff
@@ -186,6 +186,9 @@ export abstract class CreateElementTool extends PrimitiveTool {
     return super.isCompatibleViewport(vp, isSelectedViewChange);
   }
 
+  /** Convenience method to check whether control key is currently down w/o having a button event. */
+  protected get isControlDown(): boolean { return IModelApp.toolAdmin.currentInputState.isControlDown; }

```

Comment:
This seems generally useful, could it be moved to a superclass?

Reply:
Yes, I'll move to InteractiveTool and remove from CreateElementTool and ElementSetTool.